### DISCUSSION TITLE
Game crash upon starting match in multiplayer

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#15986] [Q&A] [answered]
Nothing out of the ordinary, everything was working fine until it didn't, although some minutes before that a beatmap got stuck on "importing" on my panel in multi, i instantly checked logs and it said this:

2021-12-07 17:35:18 [important]: The imported beatmap set does not match the online version.

I quit and rejoined, it was there ready to be played.
A bit later the game crashed when another map was starting.

[performance.log](https://github.com/ppy/osu/files/7670679/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7670680/runtime.log)
[database.log](https://github.com/ppy/osu/files/7670681/database.log)
[network.log](https://github.com/ppy/osu/files/7670682/network.log)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/16026
